modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -86,14 +86,13 @@ portletDisplay.setURLBack(backURL);
 
 			const postPath = scope === 'site' ? pathScopedBySite : contextPath;
 
-			let putPath = scope === 'site' ? pathScopedBySite : contextPath;
+			let patchPath = scope === 'site' ? pathScopedBySite : contextPath;
 
-			putPath = putPath.concat(
-				'/by-external-reference-code/',
-				`\${externalReferenceCode}`
+			patchPath = patchPath.concat(
+				'/<%= (objectEntry == null) ? "" : objectEntry.getId() %>'
```

Comment:
Hi @devpedrotavares,

We must continue to use the `External Reference Code` to save the entry.. 

That said, or you can try to add the `patch` [here](https://github.com/liferay/liferay-portal/blob/7b11a1fa4186195fe52164a783d3f87bca7fa98b/modules/apps/object/object-rest-impl/rest-openapi.yaml#L189-L247) and run `gw buildRest`, or you can try a different approach to fix this issue.

Please let me know if you need any assistance with this.

Thanks!